### PR TITLE
frontend: add guide section about 2FA

### PR DESCRIPTION
**Pull request by x1ddos**
This section is available while on the device settings screen.
Took the text from https://shiftcrypto.ch/start and made minor adjustments.

This is missing German translation at the moment..

R: @thisconnect

### CODE REVIEW

File: frontends/web/src/i18n/en.json [hunk 1]
```diff
@@ -648,6 +648,12 @@
                 "title": "How to securely pair with your phone",
                 "text": [ "After having downloaded our mobile app for either iOS or Android, you scan the displayed QR code, which sets up a secure channel between the mobile app and this application. Once scanned, follow the instructions in the mobile app." ]
             },
+            {   "title": "How does 2FA work",
```
Comment (KasparEtter) [inline frontends/web/src/i18n/en.json]: Move it below "What is a hidden wallet?" (since the Secrets are above Pairing).

Formulate it as a question (FAQ-like). Suggestion: "How does Two Factor Authorization (2FA) work?".

File: frontends/web/src/i18n/en.json [hunk 2]
```diff
@@ -648,6 +648,12 @@
...
+                "text": [
+                    "When 2FA is enabled, possession of a mobile phone is required in order to spend coins. Under the hood, an encrypted single-use PIN is sent to the mobile app, decrypted there, and returned to the BitBox when pressing the Accept button. This communication with the device is done via the channel between a mobile phone and this desktop app established during pairing.",
```
Comment (KasparEtter) [inline frontends/web/src/i18n/en.json]: Suggestions:
– "possession of a mobile phone is required in order to spend coins" –> "all transactions have to approved on the paired mobile phone" (I don't like the "a mobile phone" because it's a very specific one.)
– "PIN" –> "number" (I have no idea why Douglas called this PIN (also in the code). It has nothing to do with a personal identification number. Nonce and challenge would both be more suitable.)
– "between a mobile phone" –> "between the mobile phone"

File: frontends/web/src/i18n/en.json (hunk 1 above)
Comment (x1ddos) [reply frontends/web/src/i18n/en.json]: Actually, moved "What is a hidden wallet?" below "How can I disable the two factor authorization (2FA)?", so the whole list order is even more consistent imho:

- "How can I eject the BitBox?"
- "How can I eject the SD card?"
- "How to securely pair with your phone"
- "How does Two Factor Authorization (2FA) work?"
- "How can I disable the two factor authorization (2FA)?"
- "What is a hidden wallet?"

Otherwise, you'd have:

- "How to securely pair with your phone"
- "What is a hidden wallet?"
- "How does Two Factor Authorization (2FA) work?"
- "How can I disable the two factor authorization (2FA)?"

In other words, doesn't feel like "hidden wallet" belongs in between phone pairing and 2FA.

Re: title. Updated to your suggestion.

image

File: frontends/web/src/i18n/en.json (hunk 2 above)
Comment (x1ddos) [reply frontends/web/src/i18n/en.json]: All done. Thanks!

**Comment by x1ddos** [comment]
PTAL

**Comment by x1ddos** [comment]
> The hidden wallet is above 2FA in the device settings, thus I would take this entry first in the device settings

Ah, good point. Missed it. Let me quickly do that.

**Comment by x1ddos** [comment]
Done. PTAL.

image

**Comment by KasparEtter** [comment]
Great you did a test and posted it here! 🙂 Shouldn't the question at the bottom be adapted to "How can I disable Two Factor Authorization (2FA)?" then? (Or alternatively, change your new entry to lowercase?)

**Comment by x1ddos** [comment]
Yes, it should! :)
I guess I was focusing on the new stuff.
How about now:

image
